test/server/typegen.ts

```diff
@@ -30,6 +30,12 @@ test('typegen', async () => {
             }
             Relationships: []
           }
+          empty: {
+            Row: {}
+            Insert: {}
+            Update: {}
```

Comment:
These should be `Record<PropertyKey, never>` instead of `{}`, [because TypeScript™️](https://www.reddit.com/r/typescript/comments/tq3m4f/the_difference_between_object_and_recordstring/)

Reply:
Eh, probably not an issue, so nvm